Sources/LanguageServerProtocol/Requests/OpenInterfaceRequest.swift

```diff
@@ -0,0 +1,39 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2022 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+/// Request a textual interface of a module to display in the IDE.
+public struct OpenInterfaceRequest: TextDocumentRequest, Hashable {
+  public static let method: String = "textDocument/openInterface"
+  public typealias Response = InterfaceDetails
+
+  /// The document in which to lookup the module.
```

Comment:
I think a better description here would be.

```suggestion
  /// The document whose compiler arguments should be used to generate the interface.
```